js/src/Components/Widgets/LinkWidget.js

```diff
@@ -1,20 +1,44 @@
 import React, {useState} from 'react';
 import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
 import FormGroup from '@material-ui/core/FormGroup';
 import FormControl from '@material-ui/core/FormControl';
 import FormLabel from '@material-ui/core/FormLabel';
 import {FormHelperText} from "@material-ui/core";
 import Autocomplete from '@material-ui/lab/Autocomplete';
 
 export const LinkWidget = ({fieldId, defaultValue, onFieldChange, settings}) => {
-
   let timeout;
+  let initialCondition = defaultValue;
+  if (initialCondition === undefined){
+    initialCondition = [];
+    initialCondition.push(emptyLinkValue());
+  }
+
   const [urlSuggestions, setSuggestions] = useState([]);
+  const [fieldValues, setValues] = useState(initialCondition);
 
-  const defaultFieldValue = {
-    uri: defaultValue && defaultValue.length ? defaultValue[0].uri : '',
-    title: defaultValue && defaultValue.length ? defaultValue[0].title : ''
-  };
+  function emptyLinkValue() {
```

Comment:
this isn't the consistent way to define a function. and because this doesn't do anything, it could just be a `const` at the top of the component.

Reply:
I tried it as a `const` and it didn't work.  When I changed it to a function, it worked fine.  I'm not sure why that's true, but I'd love to know.

Reply:
This is now fixed.